pywemo/ssdp.py

```diff
@@ -336,16 +366,137 @@ def scan(
                     if len(entries) == max_entries:
                         return entries
     except socket.error:
-        logging.getLogger(__name__).exception(
-            "Socket error while discovering SSDP devices"
-        )
+        LOG.exception("Socket error while discovering SSDP devices")
     finally:
         for s in sockets:
             s.close()
 
     return entries
 
 
+class DiscoveryResponder:
+    """Inform Wemo devices of the pywemo virtual Wemo device.
+
+    The DiscoveryResponder informs Wemo devices of the /setup.xml URL for the
+    pywemo virtual Wemo device. The virtual device is used for receiving long
+    press actions from Wemo devices and is integrated into the
+    SubscriptionRegistry HTTP server.
+
+    Wemo devices are informed of the pywemo virtual Wemo device in two ways:
+
+    1. Wemo devices periodically send UPnP M-SEARCH discovery requests for the
+    to locate other devices on the network. DiscoveryResponder responds to these
```

Comment:
Same comment as the other PR where some of the comments, like this one, go to 80 instead of 79 characters.

Reply:
Fixed.